tests/test-with-simulated-input.el

```diff
@@ -118,10 +118,7 @@
       (let ((greeting "hello")
             (target "world"))
         (expect
-         (with-simulated-input
-             (list greeting "SPC"
-                   (list 'insert target)
-                   "RET")
+         (with-simulated-input (greeting "SPC" (insert target) "RET")
```

Comment:
`with-simulated-input` no longer supports a runtime generated KEYS list, so this test couldn't work. KEYS can contain variables, though.